packages/react/src/components/Tabs/next/Tabs.stories.js

```diff
@@ -55,18 +62,39 @@ export const Default = () => (
   </Tabs>
 );
 
-export const IconOnly = () => (
+export const Icon20Only = () => (
   <Tabs>
     <TabList aria-label="List of tabs">
-      <Tab disabled>
+      <IconTab size="lg" disabled>
         <Monster20 />
-      </Tab>
-      <Tab>
+      </IconTab>
+      <IconTab size="lg">
         <Corn20 />
-      </Tab>
-      <Tab>
+      </IconTab>
+      <IconTab size="lg">
         <Bat20 />
-      </Tab>
+      </IconTab>
+    </TabList>
```

Comment:
Would it be possible to do `<IconTabs size="lg">` so that we could define the size at the top-level of the component?

We could have the `.cds--tabs__nav-item--icon` class be something like:

```scss
.cds--tabs__nav-item--icon {
  min-height: var(--cds-tab-height, /* add whatever default height is */);
}
```

And in `tabs` set that CSS Custom Property depending on the prop passed in